### PR TITLE
GEOMESA-1839 Fix Travis CI build timeouts

### PR DESCRIPTION
**Pull request by jnh5y**
Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

File: .travisbuild.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ bash -c "while true; do sleep $PING_SLEEP; echo [INFO] \$(date -u '+%F %T UTC')
 PING_LOOP_PID=$!
 
 # build using the maven executable, not the zinc maven compiler (which uses too much memory)
-mvn clean license:check test -Phbase,travis-ci 2>&1 | tee -a $BUILD_OUTPUT | grep -e '^\[INFO\] Building GeoMesa' -e '^\[INFO\] --- \(maven-surefire-plugin\|maven-install-plugin\|scala-maven-plugin.*:compile\)'
+mvn clean license:check test -Phbase 2>&1 | tee -a $BUILD_OUTPUT | grep -e '^\[INFO\] Building GeoMesa' -e '^\[INFO\] --- \(maven-surefire-plugin\|maven-install-plugin\|scala-maven-plugin.*:compile\)'
```
Comment (elahrvivaz) [inline .travisbuild.sh]: we can remove license:check since it's part of the verify phase now

Comment (elahrvivaz) [inline .travisbuild.sh]: should we also remove the whole travis-ci profile from the pom?

**Comment by elahrvivaz** [comment]
i'll let the build finish :)
then merge

**Comment by jnh5y** [comment]
*Big money*  Err...  *Green check*  *GREEN check please!*

**Comment by elahrvivaz** [comment]
woot!

**Pull request merged**